### PR TITLE
LPS-162021 | Test Automation | Master

### PR DESCRIPTION
**Pull request by JhossephG**
https://issues.liferay.com/browse/LPS-162021

### CODE REVIEW

**Comment by JhossephG** [comment]
ci:test:sf

**Comment by epark18** [comment]
@JhossephG test steps and logic are good here! I left a comment regarding the structure of your test though when it comes to the Given, When, Then. I rewrote the test scenario on the ticket to maybe give you a better idea of it. If you need any additional clarification please let me know. Thank you!

**Comment by JhossephG** [comment]
> @JhossephG test steps and logic are good here! I left a comment regarding the structure of your test though when it comes to the Given, When, Then. I rewrote the test scenario on the ticket to maybe give you a better idea of it. If you need any additional clarification please let me know. Thank you!

Ok @epark18, I will take a look later. So,  so it's for me to change something about the "given", "when", "then" or the test structure?

**Comment by epark18** [comment]
@JhossephG Yes! You can use the Given When Then listed in the ticket and structure your test to follow that.

**Comment by JhossephG** [comment]
ci:test:sf

**Comment by epark18** [comment]
Great work, @JhossephG! It just needs that one last typo fixed in the description.

**Comment by JhossephG** [comment]
> Great work, @JhossephG! It just needs that one last typo fixed in the description.

Thank you, @epark18. But I didn't understand what is the typo in description?

File: modules/apps/object/object-test/src/testFunctional/tests/Object.testcase [hunk 1]
```diff
@@ -323,6 +323,43 @@ definition {
 			locator1 = "CreateObject#SELECT_OBJECT_FIELD_IN_TABLE");
 	}
 
+	@description = "LPS-162021 - Verify that the delete modal 'Delet' button is disabled by default"
```
Comment (timpak) [inline modules/apps/object/object-test/src/testFunctional/tests/Object.testcase:326]: @JhossephG Here is the typo, it should be `Delete` not `Delet`

Comment (JhossephG) [reply modules/apps/object/object-test/src/testFunctional/tests/Object.testcase]: My bad, I just copied and pasted the ticket description into the test and didn't see this typo. But I fixed it! :)

**Comment by epark18** [comment]
LGTM! Thank you @JhossephG

**Comment by epark18** [comment]
ci:forward

**Pull request closed without merging**